kube_metrics_server/manifest.json

```
@@ -24,7 +24,9 @@
   "integration_id": "kube-metrics-server",
   "assets": {
     "monitors": {},
-    "dashboards": {},
+    "dashboards": {
+      "Kubernetes Metrics Server Dashboard": "assets/dashboards/overview.json"
```

Shouldn't we name the dashboard `Kubernetes Metrics Server - Overview` just like the existing kube dashboards? (scheduler, controller manager..)

```suggestion
      "Kubernetes Metrics Server - Overview": "assets/dashboards/overview.json"
```